e2e/tests/rollkit/rollkit_test_util.go

```diff
@@ -0,0 +1,148 @@
+package rollkit
+
+// NOTE: this file should be able to be removed when https://github.com/strangelove-ventures/interchaintest/issues/869 is resolved
```

Comment:
:point_up:

Reply:
this is all just copied from interchaintest

Reply:
note: main now has this pulled out of the internal/ folder in main branch (not yet tagged)

Reply:
Thank you @Reecepbcups! Appreciate the update